helm/crds/application-networking.k8s.aws_accesslogpolicies.yaml

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
why we duplicate crd and template to helm? may be there is a difference that I couldn't spot

Reply:
I noticed this is how we did it with previous CRD additions, such as https://github.com/aws/aws-application-networking-k8s/pull/339/files